tasks/ModelTask.cpp

```diff
@@ -221,6 +224,17 @@ void ModelTask::updateLinks(base::Time const& time)
 {
     for(ExportedLinks::const_iterator it = exported_links.begin(); it != exported_links.end(); ++it)
     {
+        static std::map<std::string, base::Time> last_write_time;
```

Comment:
Never. *EVER* use static variables until you are *WAY* more experienced than you are.

Static variables can never be reset, are not thread-safe, and are just plain not needed.

Just store the last_update information in the internal exported_links structure.